.github/workflows/webchat-CI.yml

```diff
@@ -41,13 +40,21 @@ jobs:
       # Run a single command using the runners shell to install dependencies 
       - name: Install dependencies
         run: npm install
-      # Run linting
-      - name: Run ESLint
-        run: npm run lint --if-present
       # Get the blockedIps for this account
       - name: Get blockedIps
         run: curl -o ./src/blockedIps.json https://tl-public-chat-as-dev.s3.amazonaws.com/blockedIps.json
         shell: bash
       # Build and compile Webchat
       - name: Run build command
         run: CONFIG=as-development npm run build
+      # Run linting
```

Comment:
You could possibly also split out the linting job as a 3rd one - it would just be like the test job, just install then run the lint

Reply:
`      - name: Install dependencies
        run: npm install eslint
`
I separated the lint as a separate job and only installed eslint for this step